src/lib/ai/tools/react.ts

```
@@ -1,24 +1,53 @@
+import { createLogger } from '@/lib/logger';
 import { tool } from 'ai';
```

ESLint

[error] Unable to resolve path to module 'ai'.

(import-x/no-unresolved)